src/libstd/sys/windows/backtrace/printing/msvc.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use ffi::CStr;
+use io::prelude::*;
+use io;
+use libc::{c_ulong, c_int, c_char, c_void};
+use mem;
+use sys::c;
+use sys::dynamic_lib::DynamicLibrary;
+use sys::backtrace::BacktraceContext;
+use sys_common::backtrace::{output, output_fileline};
+
+use super::super::LOCK;
+
+type SymFromAddrFn =
+    unsafe extern "system" fn(c::HANDLE, u64, *mut u64,
+                              *mut c::SYMBOL_INFO) -> c::BOOL;
+type SymGetLineFromAddr64Fn =
+    unsafe extern "system" fn(c::HANDLE, u64, *mut u32,
+                              *mut c::IMAGEHLP_LINE64) -> c::BOOL;
+
+/// Converts a pointer to symbol to its string value.
+pub fn resolve_symname(symbol_addr: Frame, context: &BacktraceContext)
+    -> Option<&'static str>
```

Comment:
This is not documented because it's just part of the API. If you take a look at the implementation of `resolve_symname` you'll see that it creates a local `SYMBOL_INFO` on the stack. The function then returns a pointer into this structure. That'll cause segfaults on MSVC when executed.